pkg/vault/test/foundry/YieldBearingPoolBufferAsVaultPrimitive.t.sol

```diff
@@ -360,40 +343,44 @@ contract YieldBearingPoolBufferAsVaultPrimitiveTest is BaseERC4626BufferTest {
         if (vars.kind == SwapKind.EXACT_IN) {
             // Rounding issues occurs in favor of vault, and are very small
             assertLe(paths[0], vars.expectedAliceDelta, "paths AmountOut must be <= expected amountOut");
-            assertApproxEqAbs(paths[0], vars.expectedAliceDelta, 1, "Wrong path count");
+            assertApproxEqAbs(paths[0], vars.expectedAliceDelta, MAX_ERROR, "Wrong path count");
             assertLe(paths[0], vars.expectedAliceDelta, "amounts AmountOut must be <= expected amountOut");
-            assertApproxEqAbs(amounts[0], vars.expectedAliceDelta, 1, "Wrong amounts count");
+            assertApproxEqAbs(amounts[0], vars.expectedAliceDelta, MAX_ERROR, "Wrong amounts count");
             assertEq(tokens[0], address(usdc), "Wrong token for SwapKind");
         } else {
             // Rounding issues occurs in favor of vault, and are very small
-            assertGe(paths[0], vars.expectedAliceDelta, "paths AmountIn must be >= expected amountIn");
-            assertApproxEqAbs(paths[0], vars.expectedAliceDelta, 5, "Wrong path count");
-            assertGe(amounts[0], vars.expectedAliceDelta, "amounts AmountIn must be >= expected amountIn");
-            assertApproxEqAbs(amounts[0], vars.expectedAliceDelta, 5, "Wrong amounts count");
+            // TODO remove tolerance of 1 wei, when rounding issues with rates are solved
```

Comment:
Is this stale or is it a real todo?

Reply:
It's a real todo, related with the issue https://github.com/balancer/balancer-v3-monorepo/issues/884